test/mocha/27-http-edv-client-update.js

```diff
@@ -119,5 +119,39 @@ describe('bedrock-edv-storage HTTP API - edv-client update', () => {
         err.message.should.equal(
           '"sequence" is too large.');
       });
+    it('should increase sequence when updating a deleted document',
+      async () => {
+        await edvClient.insert({
+          doc: mockData.httpDocs.alpha,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        // delete doc
+        await edvClient.delete({id: mockData.httpDocs.alpha.id,
+          invocationSigner: capabilityAgent.getSigner()});
+        // get doc
+        const record = await edvClient.get({
+          id: mockData.httpDocs.alpha.id,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        record.content = {};
+        record.jwe.recipients = [];
+
+        const result = await edvClient.update({
+          doc: record,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        should.exist(result);
+        result.sequence.should.equal(2);
+
+        result.content = {};
+        result.jwe.recipients = [];
+
+        const result2 = await edvClient.update({
+          doc: result,
+          invocationSigner: capabilityAgent.getSigner(),
+        });
+        should.exist(result2);
```

Comment:
Why these extra updates -- isn't it sufficient to call `delete` and then get the document and check the sequence?

Reply:
Fixed. That was leftover from the previous attempt where deleting did not increase the sequence number but updating did.